src/main/java/study/step3/model/RacingGame.java

```diff
@@ -0,0 +1,30 @@
+package study.step3.model;
+
+import study.step3.model.car.Cars;
+
+public class RacingGame {
+
+  private Cars cars;
+
+  private int racingGameRound;
```

Comment:
```suggestion
  private final Cars cars;
  private final int racingGameRound;
```
이렇게 final로 선언해주면 어떨까요?

Reply:
racingGameRound의 경우 자동차 게임에서 고정된 숫자라 final을 붙여 재할당을 금지하는게 이해가 되는데요, Cars의 경우는 final을 붙여 Cars 객체의 재할당을 금지하는 이유가 있는건지 궁금합니다.

Reply:
반대로 재할당 하는 경우가 있을까요?